modules/combined/test/tests/gap_heat_transfer_mortar/small-2d-rz/small.i

```diff
@@ -147,6 +161,7 @@ name = 'small'
     component = x
     use_displaced_mesh = true
```

Comment:
Another follow-up change we could make if you agree is to default these "apply-contact-force" objects to use the displaced mesh. I think that'd be the right setting.